source/manual/alerts/publisher-unprocessed-fact-check-emails.html.md

```diff
@@ -0,0 +1,44 @@
+---
+owner_slack: "#govuk-2ndline"
+title: 'Publisher: Unprocessed fact-check emails'
+section: Icinga alerts
+subsection: Email alerts
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2020-07-02
+review_in: 6 months
+---
+
+### Unprocessed fact-check emails
+
+This means there are unprocessed emails in the fact-check inbox
```

Comment:
It might be worth linking to https://github.com/alphagov/publisher/blob/master/doc/fact-checking.md.